parastell/magnet_coils.py

```diff
@@ -250,23 +250,29 @@ def import_step_cubit(self):
 
         self.volume_ids = list(range(first_vol_id, last_vol_id + 1))
 
-    def export_step(self, step_filename="magnet_set", export_dir=""):
-        """Export CAD solids as a STEP file via CadQuery.
+    def export_components(
+        self, filetype="step", filename="magnet_set", export_dir=""
+    ):
```

Comment:
Would it make sense to combine `filename` and `filetype` into a single input argument? This would deviate from our established convention.